### ISSUE TITLE
Year mismatch for FEUD

### ISSUE THREAD

**Issue by pannal** [guessit-io/guessit#435]
`feud.s01e05.and.the.winner.is.(the.oscars.of.1963).720p.amzn.webrip.dd5.1.x264-casstudio.mkv` results in:

```
GuessIt found: {
    "title": "FEUD", 
    "season": 1, 
    "episode": 5, 
    "episode_title": "and the winner is", 
    "year": 1963, 
    "screen_size": "720p", 
    "streaming_service": "Amazon Prime", 
    "format": "WEBRip", 
    "audio_codec": "DolbyDigital", 
    "audio_channels": "5.1", 
    "video_codec": "h264", 
    "release_group": "casstudio", 
    "container": "mkv", 
    "mimetype": "video/x-matroska", 
    "type": "episode"
}
```

**Comment by VeNoMouS** [comment]
I wouldn't so much say its a `mismatch` , this is kind of the same issue I have with #440 , where while parsing the title AFTER the season and episode has been discovered, gets parsed, there needs to be ordering and if season and episode is discovered then do not do `year`, or `part` etc..

**Comment by pannal** [comment]
Yeah. Year for Series has to come after the series name, always. Otherwise ignore the info.

**Comment by ratoaq2** [comment]
One good thing to know is that guessit doesn't parse (or detect) the title, episode_title or alternate_title.
These properties are inferred based on the empty spaces left after all other all other properties detection.

If you have season and episode, it doesn't mean that you shouldn't have year:
`House.of.Cards.2013.S04E05.WEBRip.XviD-GROUP`

And guessit is not bound to a specific pattern as well (like scene). It's generic enough to parse and detect any kind of release name:
`House of Cards - S01 Complete - 1080p x264 ENG-ITA BluRay (2013) - GROUP`

**Comment by VeNoMouS** [comment]
@ratoaq2 that's not true if you refer #440, it clearly parses title as it cuts it up.

**Comment by ratoaq2** [comment]
Given the example:
`Adventure.Time.S08E16.Elements.Part.1.Skyhooks.720p.WEB-DL.AAC2.0.H.264-RTN.mkv`

Describing it in a simplified way:

In its first step, guessit will use all pre-configured patterns and it will detect:
```
    "season": 8, 
    "episode": 16, 
    "part": 1, 
    "screen_size": "720p", 
    "format": "WEB-DL", 
    "audio_codec": "AAC", 
    "audio_channels": "2.0", 
    "video_codec": "h264", 
    "container": "mkv", 
```

After all patterns are detected, guessit will evaluate the `holes` (portions without matches). In this case, you'll have:
- `Adventure.Time.`
- `.Elements.`
- `.Skyhooks.`
- `-RTN.`

The first portion will be inferred as `title`: Adventure Time
The second one, because it appears after episode, it will be considered as `episode_title`: Elements
The third one... guessit cannot guess, so it just leaves it.
The fourth and last one, since it's the last portion, it will be considered as `release_group`: RTN

**Comment by VeNoMouS** [comment]
But if episode and season already detected then anything after the fact should not be parsed as a field, but as title so to speak...

Addtionally, Year should before season

**Comment by pannal** [comment]
In those cases guessit seems to be in need of some tuning. 

I guess (haha), guessit v2 already does intelligent distinction between scene and non-scene naming. If not, indicators for those could be implemented fairly easily. Because at some point you have to decide which detection path you choose, otherwise it just gets too guessy (or unguessable).

Positioning seems to be a crucial refiner. Detecting anything related to the series itself (year) besides an episode title after the season/episode indices have been determined, is just pure madness.

**Comment by ratoaq2** [comment]
Fixed by #443

**Comment by VeNoMouS** [comment]
oh wow, <3 @ratoaq2 , cheers man :+1:
